Packs/GoogleThreatIntelligence/Classifiers/classifier-Google_Threat_Intelligence_DTM_Alerts-_Incoming_Mapper.json

```diff
@@ -0,0 +1,357 @@
+{
+    "description": "Map incoming DTM alert JSON data to XSOAR fields.",
+    "feed": false,
+    "id": "Google Threat Intelligence DTM Alerts - Incoming Mapper",
+    "mapping": {
+        "Google Threat Intelligence DTM Alert": {
+            "dontMapEventToLabels": false,
+            "internalMapping": {
+                "Alert ID": {
+                    "complex": {
+                        "filters": [],
+                        "root": "id",
+                        "transformers": []
+                    }
+                },
+                "GTI DTM Alert AI Doc Summary": {
+                    "complex": {
+                        "filters": [],
+                        "root": "ai_doc_summary",
+                        "transformers": []
+                    }
+                },
+                "GTI DTM Alert Aggregation ID": {
+                    "complex": {
+                        "filters": [],
+                        "root": "aggregated_under_id",
+                        "transformers": []
+                    }
+                },
+                "GTI DTM Alert Analysis": {
+                    "complex": {
+                        "filters": [],
+                        "root": "analysis",
+                        "transformers": []
+                    }
+                },
+                "GTI DTM Alert Attachments": {
+                    "complex": {
+                        "filters": [],
+                        "root": "attachments",
+                        "transformers": []
+                    }
+                },
+                "GTI DTM Alert Confidence Score": {
+                    "complex": {
+                        "filters": [],
+                        "root": "confidence",
+                        "transformers": [
+                            {
+                                "args": {
+                                    "by": {
+                                        "isContext": false,
+                                        "value": {
+                                            "simple": "2"
+                                        }
+                                    }
+                                },
+                                "operator": "precision"
+                            }
+                        ]
+                    }
+                },
+                "GTI DTM Alert Created At": {
+                    "complex": {
+                        "filters": [],
+                        "root": "created_at",
+                        "transformers": []
+                    }
+                },
```

Comment:
The 'GTI DTM Alert Doc Description' field is mapped with a root of 'description'. However, in the provided test data, 'description' is nested under 'doc'. Please adjust the mapping to 'root: doc, accessor: description'.